### ISSUE TITLE
Media presence element on search page lacks an accessible name

### ISSUE THREAD

**Issue by BuddyinKC** [omeka-s-themes/freedom#60]
We have two exhibits using this theme and DubBot flagged both search pages as the Search for media presence select element appears to lack an associated label. Here are the two pages: https://exhibits.library.umkc.edu/s/antiguedades-mexicanas/item/search and https://exhibits.library.umkc.edu/s/mcilrath-swhof/item/search

**Comment by aviannamiller** [comment]
Thank you for bringing this up. This was fixed in https://github.com/omeka-s-themes/freedom/issues/48 and included in the latest release, [v1.1.0](https://github.com/omeka-s-themes/freedom/releases/tag/v1.1.0).
